src/tutorial/tutorial.cpp

```diff
@@ -35,6 +37,8 @@ void Tutorial::allowItem(const QString& objectName) {
   m_allowedItems.append(objectName);
 }
 
+QString Tutorial::currentTutorialId() { return m_currentTutorial->id(); }
```

Comment:
1. What about m_currentTutorial is null?
2. `Tutorial::currentTutorialId() const`

Reply:
Will remove this in favor of the `Q_PROPERTY`